hivemq-edge/src/frontend/src/extensions/datahub/components/nodes/CustomHandle.tsx

```diff
@@ -0,0 +1,52 @@
+import { CSSProperties, FC, HTMLAttributes, useMemo } from 'react'
+import { getConnectedEdges, Handle, HandleProps, useNodeId } from 'reactflow'
+
+import useDataHubDraftStore from '../../hooks/useDataHubDraftStore.ts'
+
+interface CustomHandleProps extends Omit<HandleProps & Omit<HTMLAttributes<HTMLDivElement>, 'id'>, 'isConnectable'> {
+  isConnectable?: boolean | number
+}
+
+export const CustomHandle: FC<CustomHandleProps> = (props) => {
+  const { nodes, edges } = useDataHubDraftStore()
+  const nodeId = useNodeId()
+
+  const isHandleConnectable = useMemo(() => {
+    if (typeof props.isConnectable === 'number') {
+      const node = nodes.find((e) => e.id === nodeId)
```

Comment:
question(non-blocking): I'm a big fan of preventing abbreviations and use the sigular of a list
It becomes easier to parse. As an example I first thought it was an `event` then `error` then I saw it's a filter.

```suggestion
      const node = nodes.find((node) => node.id === nodeId)
```

Reply:
Lol, that's `e` for element!
It's interesting how a good IDE changes practices, as the implicit typing is always visualised on my editors so I don't have the confusion for the small internals like these.

![Screenshot 2024-01-25 at 09 54 58](https://github.com/hivemq/hivemq-edge/assets/2743481/5771d6d7-89fe-4841-a77e-81d4731a3bb9)

Point taken will pay more attention

Reply:
Ah that make sense! I like to review within GitHub. That one does sadly not include the Type inference. Thank you for taking care 👍